.eslintrc.js

```diff
@@ -26,6 +26,15 @@ module.exports = {
         ecmaVersion: 2018
     },
     rules: {
-        semi: ["error"]
+        semi: ['error'],
+        indent: ['error', 4, { SwitchCase: 1 }],
+        'prefer-const': ['error'],
+        'no-var': ['error'],
+        'prefer-destructuring': ['error'],
+        'object-shorthand': ['error'],
+        'object-curly-spacing': ['error', 'always'],
+        'quotes': ['error', 'single'],
```

Comment:
You don't need to quote this key.

Reply:
I've added `quote-props` to enforce this (using the "as-needed" option): https://eslint.org/docs/rules/quote-props